cloudprober/templates/deployment.yaml

```diff
@@ -12,6 +12,9 @@ spec:
   template:
     metadata:
       annotations:
+        {{- if $.Values.podAnnotations }}
+          {{ toYaml .Values.podAnnotations | nindent 8 }}
```

Comment:
Minor change to kill any spaces in the beginning.

```suggestion
          {{- toYaml .Values.podAnnotations | nindent 8 }}
```

Reply:
Sure yes suggestion committed 👍